misc.go

```diff
@@ -22,8 +22,9 @@ import (
 
 // SlackResponse handles parsing out errors from the web api.
 type SlackResponse struct {
-	Ok    bool   `json:"ok"`
-	Error string `json:"error"`
+	Ok               bool                   `json:"ok"`
+	Error            string                 `json:"error"`
+	ResponseMetadata map[string]interface{} `json:"response_metadata"`
```

Comment:
Maybe you can use `struct { Message []string }` as the type of `response_metadata` instead of `map[string]interface{}`.

e.g. official Java Implementation: https://github.com/slackapi/java-slack-sdk/blob/main/slack-api-model/src/main/java/com/slack/api/model/ResponseMetadata.java

Reply:
hey @kanata2 thanks for your feedback! I would be open to using a struct instead of a map here, will update!